validation/api/api.lock.json

```diff
@@ -900,8 +900,8 @@
     "exports": [
       "public class AsyncRandom(generator : () -> async* Blob)",
       "public let blob : shared () -> async Blob",
-      "public func new(seed : Blob) : Random",
-      "public func newAsync() : AsyncRandom",
+      "public func crypto() : AsyncRandom",
+      "public func fast(seed : Nat64) : Random",
```

Comment:
Would better names be cryptographic/pseudo?

Reply:
I considered `pseudo` in place of `fast` but thought that `fast` might be a more immediately clear name, i.e., instead of `pseudo` we would probably just want to call it `sfc64()`.

We could perhaps rename `crypto()` to `cryptographic()`. The reasoning for `crypto()` is that it implies that you want to use this RNG for cryptography, cryptocurrencies, etc. in the theme of naming based on the intended use case. I think `crypto` is an acceptable shorthand for `cryptographic` in this context but would be open to other opinions on that.